src/sagemaker/jumpstart/estimator.py

```diff
@@ -712,13 +712,26 @@ def attach(
         Returns:
             Instance of the calling ``JumpStartEstimator`` Class with the attached
             training job.
+
+        Raises:
+            ValueError: if the model ID or version cannot be inferred from the training job.
+
         """
 
         if model_id is None:
-            model_id, inferred_model_version = get_jumpstart_model_id_version_from_training_job(
-                training_job_name=training_job_name,
-                sagemaker_session=sagemaker_session,
+
+            region: str = sagemaker_session.boto_region_name
+            partition: str = aws_partition(region)
+            account_id: str = sagemaker_session.account_id()
+
+            training_job_arn = (
+                f"arn:{partition}:sagemaker:{region}:{account_id}:training-job/{training_job_name}"
```

Comment:
question: does the training job name need to be lower cased?

Reply:
good point, let me check

Reply:
It should not be lower cased:

```
dev-dsk-evakravi-1e-d82d1ee5 % aws sagemaker list-tags    --resource-arn  arn:aws:sagemaker:us-west-2:802376408542:training-job/inference-japanese-stablelm-instruct-alph-20231114-1448-none-a0

An error occurred (ValidationException) when calling the ListTags operation: Resource does not exist.
(env) 
(23-12-27 20:03:56) <255> [~/workspace/github/sdk-devx-improvements-2/sagemaker-python-sdk]  
dev-dsk-evakravi-1e-d82d1ee5 % aws sagemaker list-tags    --resource-arn  arn:aws:sagemaker:us-west-2:802376408542:training-job/inference-japanese-stablelm-instruct-alph-20231114-1448-None-a0
{
    "Tags": [
        {
            "Value": "ml.p3.2xlarge", 
            "Key": "modelhub-instance-type"
        }, 
        {
            "Value": "huggingface-textgeneration1-japanese-stablelm-instruct-alpha-7b-v2-fp16", 
            "Key": "modelhub-model-id"
        }, 
        {
            "Value": "inference-japanese-stablelm-instruct-alph-20231114-1448-None-a0", 
            "Key": "modelhub-transfer-learning-job-name"
        }
    ]
}
```